backend/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+
+  mysqldb:
+    image: mysql:8
+    container_name: mysqldb
+    ports:
+      - 3305:3306
```

Comment:
You can use the same port. Docker has its internal file system and networking so there won't be any interference.

Reply:
All of us on the backend have MySQL already installed, so we all have a service running on 3306.
Mapping Docker's internal port 3306 to 3305 seemed to be the easiest solution.

Reply:
Still, the app will communicate with Docker 3306 port, you via DB client will communicate with 3306 port from your local machine. The Docker 3306 port will communicate with 3306 from your local machine, it may be set as the same port in value, but it is not the same port logically.

Reply:
Running it with
`    ports:
      - 3305:3306`
results in an error:
`Error response from daemon: driver failed programming external connectivity on endpoint mysqldb (cb1cad26117a0961d1fd3fc20cbd59443c13382711161a030546dd0c10c333fc): Error starting userland proxy: error while calling PortManager.AddPort(): listen tcp4 0.0.0.0:3306: bind: address already in use`

This is why I put 3305. Do I need to change something else as well for it to work with 3306?

Reply:
I see the problem now, I'll change it to 3306. Thanks for the help.